frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -51,28 +63,35 @@ const InputForm = () => {
   const [jobs, setJobs] = useState<JobCategoryInterface[]>([
     { id: 0, name: "API Unavailable" },
   ]);
+  const [skills, setSkills] = useState<string[]>([]);
+  const [autoCompleteFocus, setAutoCompleteFocus] = useState<boolean>(false);
 
   const formik = useFormik({
     initialValues: {
       industryCategory: "",
       yearsOfExperience: "",
       city: "",
-      relevantSkills: "",
+      relevantSkills: [] as string[],
       academicCredentials: "",
       numberOfSearchResults: NumberOfSearchResultsOptions.Option1,
     },
     validationSchema: validationSchema,
-    onSubmit: (values) => {
-      alert(JSON.stringify(values, null, 2));
+    onSubmit: async (values) => {
       setSearchValues({
         industryCategory: formik.values.industryCategory,
         yearsOfExperience: formik.values.yearsOfExperience,
         city: formik.values.city,
-        relevantSkills: formik.values.relevantSkills,
+        relevantSkills: formik.values.relevantSkills.join(", "),
         academicCredentials: formik.values.academicCredentials,
         numberOfSearchResults: `${formik.values.numberOfSearchResults.toString()}`,
       });
-      navigate("/results");
+
+      try {
+        await axios.post(`${baseBackendUrl}/api/job/results`, values);
```

Comment:
why is this post request being made? not sure this post request does anything

Reply:
I was using the /api/job/results post to send the form data to our backend. The updated app.py in this PR uses this post request to direct traffic from the input form to the model. Here's an example:
![image](https://github.com/user-attachments/assets/a9412579-3401-49be-a5cb-beea8cf41030)

Reply:
sure, i understand that this is creating a post request, but it's not actually doing anything with the response from the post request. There's a post request that is actually run in `SearchResults.tsx`. I realize this probably isn't ideal, and the post request should be on the form page, but if we put it in the form page, the results page has no way to access the data from the API.

Reply:
ah, I see what you mean. I'll remove it.